src/js/services/pinterest.js

```diff
@@ -9,7 +9,7 @@ module.exports = function(shariff) {
         title += ' - ' + creator;
     }
     var img = shariff.getOption('mediaUrl');
-    if (img && img.length <= 0) {
+    if (!img || img.length <= 7) {
```

Comment:
Shouldn't this be at least 10 characters? I think `http://a.b` makes more sense as a minimum URL than `http://`.